configs/ext/tfm_mbedcrypto_config_profile_medium.h

```diff
@@ -234,7 +234,7 @@
  * mechanism). You may still freely enable additional `MBEDTLS_XXX` symbols
  * in mbedtls_config.h.
  *
- * If the symbol #MBEDTLS_PSA_CRYPTO_CONFIG_FILE is defined, it specifies
+ * If the symbol #TF_PSA_CRYPTO_CONFIG_FILE is defined, it specifies
```

Comment:
I forgot about it previously but we need to update configs/ext/README.md explaining the current situation where the TF-M config files are not directly from TF-M but have been adapted to the Mbed TLS configuration file split and more generally to the Mbed TLS repository split.